conf/proxy.conf

```diff
@@ -49,6 +49,9 @@ authenticationProviders=
 # Enforce authorization
 authorizationEnabled=false
 
+# Authorization provider name list, which is comma separated list of class names
+authorizationProviders=org.apache.pulsar.broker.authorization.PulsarAuthorizationProvider
```

Comment:
What is the behavior when multiple backends are configured? at-least-once has to say "yes"? It might get tricky to define well, when there are multiple source of truth that we're following.

Reply:
if authMethod is not provided by client then [AuthorizationService](https://github.com/rdhabalia/pulsar/blob/23181386aafd89255823fa33138cbbe68c7d835c/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/AuthorizationService.java#L270) iterates over all the providers and look for one "yes". 

However, right now, if client doesn't provide authMethodName then it I have kept [default](https://github.com/rdhabalia/pulsar/blob/23181386aafd89255823fa33138cbbe68c7d835c/pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java#L71) authMethod in [protoAPI](https://github.com/rdhabalia/pulsar/blob/23181386aafd89255823fa33138cbbe68c7d835c/pulsar-common/src/main/proto/PulsarApi.proto#L156) which makes broker to do authorization against default-existing-provider to avoid iteration over all available providers as all old client don't send authMethod.

Reply:
I think it might not be always desirable to have the client choose which authorization backend to use. Do you have a concrete use case example to have multiple auth backends? I would actually just vote to have a single backend impl allowed. :)

Reply:
I don't have a strong opinion for having multiple providers but it can be useful when one wants to manage multiple authorization-systems for time being when they are migrating from one authorization provider to another. I agree that it's not desirable that client chooses the auth-method. I will remove support of multiple providers.

Reply:
Ok, it is always easier to add it if needed, then remove it later